modules/ROOT/pages/values-and-types/temporal.adoc

```diff
@@ -1139,13 +1140,16 @@ RETURN d.minutesOfHour, d.secondsOfMinute, d.millisecondsOfSecond, d.microsecond
 5+d|Rows: 1
 |===
 
-======
+The returned values all count towards the next full hour, minute or second respectively.
+For example, `d.microsecondsOfSecond` has a value of 111111 because it's the 111111111 nanoseconds from the query in microseconds (rounded down) but it's not another full second.
```

Comment:
```suggestion
For example, `d.microsecondsOfSecond` has a value of `111111` because it is the `111111111` nanoseconds from the query in microseconds (rounded down) but it is not another full second.
```